app/models/manageiq/providers/vmware/infra_manager/ems_ref_obj_mixin.rb

```diff
@@ -1,6 +1,8 @@
 module ManageIQ::Providers::Vmware::InfraManager::EmsRefObjMixin
   extend ActiveSupport::Concern
 
+  autoload :VimString, 'VMwareWebService/VimTypes'
```

Comment:
NOTE the specs were failing here and this seemed like a better fix than adding a require in the spec